aries-site/src/pages/foundation/voice-and-tone.mdx

```diff
@@ -1 +1,360 @@
+
+Unlike voice, tone can change based on the target audience and the subject matter. It is situational. For example, tone is very different when giving directions during an emergency vs ordering a pizza.
+
+### Point of View
+
+Point of view has to do with the perspective from which the message is written. First person perspective uses words like: I, me, us, we, our, etc. Second person uses any form of the word "you" which has the effect of addressing the reader. Third person directs the reader to things that are neither the writer or the reader.
+
+Examples:
+
+- *We noticed the server is malfunctioning.* (1st person-ish)
+- *Your server is malfunctioning.* (2nd person-ish)
+- *The server is malfunctioning.* (3rd person - no focus on the writer or the reader)
+
+### Our Voice and Tone
+
+We want our messaging to be informative, concise, and trustworthy. We should be perceived as intelligent, experienced, inclusive and non-threatening. Our messaging should be active, positive, objective, conversational American English written in a third person point of view.
+
+The following are some high-level guidelines for constructing good messaging:
+
+- Conversational American English
+- Correct grammar and punctuation
+- Active tense
+- Positive, not negative
+- Not robotic sounding
+- Never condescending
+- Establish context
+- Lead with what is most important
+- Inspire trust and confidence
+- Explain using terminology consistent with the UI
+- Avoid charged words like "*error*"
+- Avoid blame casting (on user or on software)
+- State facts without coloring them in opinions
+- Avoid personal pronouns
+- Careful with words like "*Please*"
+- Do not use "*contact your administrator*" (the admin is likely the user) or "*contact HPE*" (HPE might not be the support contact -- it might be a channel partner)
+- Read your message out loud to another person. Is this how you would say it out loud? Does the other person feel this is how two people would converse?
+
+## Capitalization
+
+The general rule is that things that are sentences should follow sentence capitalization rules. And, things that are titles/headings/labels should follow title capitalization rules.
+
+Sentences follow sentence capitalization rules:
+
+- Paragraphs of text
+- Tooltips
+- Notifications
+- Help
+
+Titles follow title capitalization rules:
+
+- Page/dialog titles
+- Section headings
+- Attribute labels
+- Table column headings
+- Legends
+- Tabs/Buttons
+
+Capitalize proper nouns. HPE brand and product names are proper nouns.
+
+Fight the urge to elevate normal nouns to proper noun status. For example, if you are in a "data sources" page and need to say something about data sources, do not capitalize it. Unless it is a true proper noun, leave it uncapitalized when following sentence capitalization rules.
+
+DO NOT USE ALL CAPS IN EITHER TITLES OR SENTENCES. It is harder to read and is equated with yelling.
+
+## Acronyms
+
```

Comment:
```suggestion

Acronyms and initialisms can be problematic and should be used with care. 
Because they are shortcuts communicating a concept in a visually shortened form, they 
tend to become distinct languages for a limited set of "insiders." 
However, for those who are less fluent or use the language infrequently, 
acronyms and initialisms are actually detrimental. Acronyms increase the cognitive load 
on the user, forcing them to "decode" the message before comprehension, and tend to 
be exclusionary rather than inclusive.
```